app/components/ElectronicPharmacyStockRecordImportWizard.js

```diff
@@ -185,8 +185,8 @@ class ElectronicPharmacyStockRecordImportWizard extends Component<
                   dataRowsFrom
                 ) ||
                   headerRow < 0 ||
-                  dataRowsFrom <= headerRow ||
-                  dataRowsTo <= dataRowsFrom))
+                  dataRowsFrom < headerRow ||
```

Comment:
I'm doubtful about changing the comparison between `dataRowsFrom` and `headerRow`. Even though Manuel wrote "a user may want to upload a file with only one row", I think he was referring to a data row (in addition to the header). It doesn't make much sense to me uploading a file with just headers. So this clause of the comparison should remain the same.  Let's ask him and see what he says , what do you think?

Reply:
I think you're interpreting the code in a weird way - let me walk you through it because I'm not sure I fully get it anyway 😅 it's a confusing bit

What this specific line used to do was disable the "next" button if, for example, the header row was 2 and the starting row was 1 or 2. With the new code, for the same example it only gets disabled for the case of starting row 1, because you could theoretically upload a file with just one row of data (as far as I understood).

Reply:
Ok. I'm interpreting this in this way (I'll use the same values):

Current behaviour:
`Header row is 2`. So starting row has to be `1` or `2`. That's ok, because my data has to start at least at row 3.
But what I can't do now is setting `dataRowsFrom = dataRowsTo = 3` (in case I have just one data row)

So IMO not allowing `dataRowsFrom <= headerRow` should remain that way. But it's still a domain question

Reply:
But you CAN set `dataRowsFrom = dataRowsTo = 3`! Check out line 189!

Reply:
Yepp. Change in line 189 is correct and is required to solve the issue👍. Not sure about change in line 188, it depends on what "data row" means and if the header row is considered or not a data row